frontend/components/Chart/PiChartView.ts

```diff
@@ -63,22 +68,19 @@ export class PiChartView extends View {
 
   renderPiChart(arr) {
     this.$piChart.innerHTML = ''
+    this.$piTable.innerHTML = ''
+    const totalAmount = arr.reduce((a, b) => a + b.amount, 0)
```

Comment:
```suggestion
    const totalAmount = arr.reduce((sum, item) => sum + item.amount, 0)
```
a, b보다 의미있는 변수명이 좋을 것 같아요!